src/main/java/uk/gov/hmcts/reform/pbis/servicebus/IServiceBusClient.java

```diff
@@ -2,11 +2,19 @@
 
 import com.microsoft.azure.servicebus.IMessage;
 
+import java.util.Map;
 import java.util.UUID;
 
 public interface IServiceBusClient extends AutoCloseable {
 
     IMessage receiveMessage();
 
     void completeMessage(String messageId, UUID messageLockToken);
+
+    void sendToDeadLetter(
+        IMessage message,
+        String reason,
```

Comment:
`reason` and `description` correspond to `DeadLetterReason` and `DeadLetterErrorDescription` - properties that are also populated by Service Bus. Reason is more like an enum, while description provides an explanation.